diff --git a/Android.bp b/Android.bp
index 4bcdbd5..bd5d904 100644
--- a/Android.bp
+++ b/Android.bp
@@ -39,7 +39,7 @@ rust_library {
     host_supported: true,
     crate_name: "grpcio_sys",
     cargo_env_compat: true,
-    srcs: ["src/lib.rs"],
+    srcs: ["src/lib.rs", ":libgrpc_bindgen"],
     edition: "2018",
     features: [
         "_gen-bindings",
@@ -48,6 +48,7 @@ rust_library {
         "boringssl",
         "boringssl-src",
     ],
+    cfgs: ["soong"],
     rustlibs: [
         "liblibc",
         "liblibz_sys",
