diff --git a/grpc/third_party/abseil-cpp/absl/strings/internal/str_format/extension.h b/grpc/third_party/abseil-cpp/absl/strings/internal/str_format/extension.h
index 55cbb56..b696c4f 100644
--- a/grpc/third_party/abseil-cpp/absl/strings/internal/str_format/extension.h
+++ b/grpc/third_party/abseil-cpp/absl/strings/internal/str_format/extension.h
@@ -19,6 +19,8 @@
 #include <limits.h>
 
 #include <cstddef>
+// Added to fix compilation errors as it was missing. Should be fixed upstream.
+#include <cstdint>
 #include <cstring>
 #include <ostream>
 
